shared/studio/components/visualQuerybuilder/querybuilder.module.scss

```diff
@@ -193,6 +194,31 @@
   }
 }
 
+.checkbox {
+  margin-right: 6px;
+  width: 12px;
+  height: 12px;
+  cursor: pointer;
+  &:after {
+    content: " ";
+    display: inline-block;
+    position: relative;
+    width: 12px;
+    height: 12px;
+    margin-right: 4px;
+    background: var(--app-panel-bg);
+    border: 1px solid var(--app-text-colour);
+    border-radius: 2px;
+  }
```

Comment:
Unfortunately this only works in webkit based browsers. The others don't support pseudo elements on input controls like checkboxes. I think using `appearance: none` to remove the native checkbox style and then applying the styles in the `:after` directly, is currently the best cross-browser way of styling checkboxes. There's an example that used here in the data editor: https://github.com/edgedb/edgedb-ui/blob/main/shared/studio/tabs/dataview/dataInspector.module.scss#L581-L587

Reply:
Hey tnx, this is much better approach. I usually try to avoid before/after because it is not great for accessibility but this time didn't come up with better solution. `appearance: none`is great, tnx